reload_host_config.sh

```diff
@@ -9,8 +10,28 @@ then
     echo "Example:"
     echo "$0 enp0s4"
 else
+    # if the parameter is present, cache root credentials
+    sudo -v
+    if [ $? == 1 ]
+    then
+        echo "[ERRO] Without root permission, you cannot change iptables configuration"
+        exit 1
+    fi
+    # if user has root permissions, then start to modify the rules
     echo "[INFO] Using $1 as interface name"
 
+    # first, delete any previous applied rules
+    echo -n "[INFO] Removing all old iptables rules, if any... "
```

Comment:
The iptables rules should probably not be all cleaned, it could cause issues
You should only delete ones created by the very same script, maybe by creating a dedicated chain?

Reply:
Hello, you have made a good point. As I'm not an expert on iptables chains, IG I'll be adding a parameter so the user trigger this "reset" by his own will, instead of doing it automatically by default on the script and without any warning. I'll work on that on the coming days and update this PR.

Reply:
So, now the workflow is updated to solve the concern raised

BTW, @linouxis9 thanks for you comment

I'll make this PR ready to review again but if any other adjustments are needed before merging, don't hesitate to tell me